tests/cgroup

```diff
@@ -12,6 +12,13 @@ lxd init --auto
 
 IMAGE="${TEST_IMG:-ubuntu-minimal-daily:24.04}"
 
+rc=0
+journalctl --quiet --no-hostname --no-pager --boot=0 --unit=snap.lxd.daemon.service | grep "Kernel supports swap accounting" || rc="$?"
+LXCFS_SUPPORTS_SWAP_ACCOUNTING=0
+if [ "${rc}" -eq 0 ]; then
```

Comment:
```suggestion
LXCFS_SUPPORTS_SWAP_ACCOUNTING=0
if journalctl --quiet --no-hostname --no-pager --boot=0 --unit=snap.lxd.daemon.service --grep "Kernel supports swap accounting"; then
```

Out of curiosity, should my 6.8 (Jammy HWE) kernel meet the needed requirements for this? I don't have that message with a fresh `latest/edge` snap:

```
$ snap list lxd
Name  Version      Rev    Tracking     Publisher   Notes
lxd   git-538cfa3  30507  latest/edge  canonical✓  -
$ uname -a
Linux sdeziel-lemur 6.8.0-45-generic #45~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Wed Sep 11 15:25:05 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
$ cat /proc/cmdline 
BOOT_IMAGE=/vmlinuz-6.8.0-45-generic root=UUID=78a3536d-545e-4ccd-8c89-3e5165da9420 ro rootflags=subvol=@ quiet splash debugfs=off nmi_watchdog=0 nr_cpus=12 panic=300 8250.nr_uarts=1 vsyscall=none vt.handoff=7
```

Reply:
It's not really about kernel. This specific check is about LXCFS. The problem is that currently LXCFS is broken on cgroup2 and always says that `Kernel does not support swap accounting` while it is supported by the kernel.

On `latest/edge` you should get LXCFS from the `main` branch where it's fixed by https://github.com/lxc/lxcfs/pull/663

Reply:
To conclude, if you have:
- recent LXD from `latest/edge`
- kernel 6.8
- cgroup2

it should say `Kernel supports swap accounting` in lxd daemon logs

Reply:
OK I tick all the boxes but I had not started LXD after the refresh... my bad. Thanks!